app/assets/stylesheets/themes/cultural_repository.scss

```diff
@@ -303,16 +295,20 @@
     body.dashboard {
       padding-top: 0 !important;
     }
+  }
+
+  @media (max-width: 992px) {
+    #facets .top-card-heading {
+      border: 1px solid #fff;
+      background-color: #fff;
+    }
 
-    .facet-panel-background-color {
+    #facets .facet-panel-background-color {
```

Comment:
The cultural repository homepage does NOT use the facet panel color set in the appearance colors settings... this is supposed to override it and no longer does.

Reply:
We determined that when an admin changes the facet colors in the Cultural Theme, this should update the facets in the header and on the search page. Previously, this behavior was not happening in production leading to some confusion around the desired behavior. As of this PR the colors will update when an admin sets alternative colors for the facets in the Cultural Theme.